website/content/en/docs/usage/operator-metrics.md

```diff
@@ -7,56 +7,72 @@ description: >
   How to monitor and consume metrics emitted by CAP Operator
 ---
 
-The [Controller](docs/concepts/operator-components/controller/) and [Subscription Server](docs/concepts/operator-components/subscription-server/) now emit [Prometheus metrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus) at `/metrics` path and `9090` port that may be used by consumers to analyze and understand usage, detect potential issues, monitor and scale cluster resources.
-You can enable
+## Overview
 
-### Controller metrics
-The controller emits [standard go metrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/collectors#WithGoCollectorRuntimeMetrics), workqueue metrics for the resources being reconciled implemented based on [MetricsProvider](https://pkg.go.dev/k8s.io/client-go/util/workqueue#MetricsProvider) and the following additional metrics:
+The CAP Operator includes built-in [Prometheus metrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus) that enable users to effectively monitor and analyze the operator's performance. These metrics can provide insights into resource usage, potential issues, and overall operator health. The metrics are accessible at the `/metrics` endpoint on port `9090` of both the [Controller](docs/concepts/operator-components/controller/) and the [Subscription Server](docs/concepts/operator-components/subscription-server/).
 
-{{% pageinfo %}}
+## Controller Metrics
+
+The Controller emits several types of metrics, including:
+
+- **Standard Go Metrics**: These metrics are provided by the [Prometheus Go client](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/collectors#WithGoCollectorRuntimeMetrics) and include runtime statistics.
+- **Workqueue Metrics**: These metrics are relevant to the resources being reconciled and are based on the [MetricsProvider](https://pkg.go.dev/k8s.io/client-go/util/workqueue#MetricsProvider).
+- the following specific metrics:
+
+### Specific Metrics
+
+1. **Reconcile Errors**
 ```
 cap_op_reconcile_errors{kind="CAPApplication",name="my-app",namespace="app"} 11

```

Comment:
```suggestion
cap_op_reconcile_errors{kind="CAPApplication",name="my-app",namespace="app"}
```
Not sure if it is a typo or not. I can see a number in all metrics expressions

Reply:
The whole expression is s sample including labels and numbers